src/langcheck/metrics/de/reference_free_text_quality.py

```diff
@@ -316,172 +226,99 @@ def fluency(generated_outputs: List[str] | str,
-    score. If a score could not be computed, `None` is inserted to the score
-    and explanation lists.
+    generated outputs using the provided EvalClient, using a prompt that is
+    similar to the one used in G-Eval (see the Ref below). This metric takes on
+    float values that are either 0, 0.5, or 1, where 0 is "poor" fluency, 0.5 is
+    "fair" fluency, and 1 is "good" fluency.  If a score could not be computed,
+    `None` is inserted to the score and explanation lists.
 
     Ref:
         https://github.com/nlpyang/geval/blob/main/prompts/summeval/flu_detailed.txt
-        https://platform.openai.com/docs/guides/gpt/function-calling
 
     Args:
         generated_outputs: A list of model generated outputs to evaluate
-        client_type: The type of OpenAI client ('openai' or 'azure_openai')
-        client: (Optional) OpenAI or AzureOpenAI client. If this is None, we
-            will attempt to create a default client depending on the
-            ``client_type``.
-        openai_args: (Optional) Dict of additional args to pass in to the
-            ``client.chat.completions.create`` function
-        use_async: Whether to use the asynchronous API of OpenAI
+        eval_client: EvalClient instance used to evaluate the generated outputs
 
     Returns:
         score_list: a list of scores
         explanation_list: a list of explanations for the scores
     '''
-
-    def _prompt(gen_output: str) -> str:
-        return f'''
-        Sie bewerten die Flüssigkeit einer eingereichten Aussage. Hier sind die
-        Daten:
-        [BEGINN DATEN]
-        ************
-        [Benutzeranfrage]: {gen_output}
-        ************
-        [ENDE DATEN]
-
-        Bestimmen Sie die Flüssigkeit der eingereichten Aussage. Die verfügbaren
-        Bewertungen sind:
-        `Schlecht` - Die Aussage hat viele Fehler, die sie schwer verständlich
-        oder unnatürlich wirken lassen.
-        `Ausreichend` - Die Aussage hat einige Fehler, die die Klarheit oder
-        Flüssigkeit des Textes beeinträchtigen, aber die Hauptpunkte sind
-        dennoch verständlich.
-        `Gut` - Die Aussage hat wenige oder keine Fehler und ist leicht zu
-        lesen undmzu verstehen.
-
-        Atmen Sie tief durch und bearbeiten Sie dieses Problem Schritt für
-        Schritt.
-        '''
-
-    def _function_call_prompt(long_assessment: str) -> str:
-        return f'''
-        Folgendes ist eine Bewertung zur Sprachflüssigkeit einer Aussage:
-        ************
-        [Bewertung]: {long_assessment}
-        ************
-
-        Speichern Sie die resultierende Bewertung. Die verfügbaren Bewertungen
-        sind:
-        `Schlecht`
-        `Ausreichend`
-        `Gut`
-        '''
+    fluency_template = get_template('de/metrics/fluency.j2')
 
     fluency_assessment_to_score = {
-        'Schlecht': 0,
-        'Ausreichend': 0.5,
-        'Gut': 1.0,
+        'Poor': 0,
+        'Fair': 0.5,
+        'Good': 1.0,
     }
```

Comment:
No, I did that for consistency. All the Japanese prompts use English options, and because of the non-ascii characters I needed to replace some German options to English too. It is weird if the language used as the options changes based on whether the characters like "ä", "ö" are used or not, so I replaced all of them with English words.

Reply:
Ok sgtm! Yeah I changed the Japanese options to English ones because OpenAI sometimes went haywire. As long as the descriptions of the options remain in the same language it seems like it works fine